docs/installation/tango.md

```diff
@@ -77,7 +77,13 @@ This guide shows how to setup Tango in a **development environment**. Use the [p
         :::bash
         cp config/autogradeConfig.rb.template config/autogradeConfig.rb
 
-    Fill in the correct info for your Tango deployment, mainly the following:
+    This file contains declarations to autograder constants. However, many of these constants by default reference an
```

Comment:
I split up the creation of the .env file in the Autolab installation page, but I will still leave the steps to fill in the .env file in the Tango installation steps as well, since it's more likely to fill in the .env file when installing Tango, which would usually come after installing Autolab.